v21.1/restore.md

```diff
@@ -442,19 +424,9 @@ To view the available subdirectories, use [`SHOW BACKUPS`](#view-the-backup-subd
 
 Restoring from [incremental backups](take-full-and-incremental-backups.html#incremental-backups) requires previous full and incremental backups. To restore from a destination containing the full backup, as well as the incremental backups (stored as subdirectories):
```

Comment:
might be worth adding "requires full and incrmental backups [ to live in the same subdirectory]"

and then deleting the second sentence.

Reply:
+1